doc/strace.1.in

```diff
@@ -57,6 +57,7 @@ strace \- trace system calls and signals
 .OP \-U columns
 .OP \-X format
 .OP \-\-seccomp\-bpf
+.if '@ENABLE_STACKTRACE_FALSE@'#' .OP \-\-stacktrace\-frames\-limit\fR=\fIlimit\fR
 .OP \-\-syscall\-limit limit
```

Comment:
There seems to be a little inconsistency with the naming.  From one side, we have --syscall-limit and --string-limit options.  From another side, here we have --stacktrace-frame**s**-limit.  Also, it's not clear why do we have `stack trace` concatenated into a single word.